### PR TITLE
CMakeList: Fix build with CMAKE VERSION < 3.1

### PR DESCRIPTION
**Pull request by pudding8757**
C_STANDARD was introduced since cmake version 3.1 so it might not work
with old build tools

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -53,7 +53,13 @@ if(NOT DEFINED CMAKE_INSTALL_INCLUDEDIR)
 endif()
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-set(CMAKE_C_STANDARD 99)
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
```
Comment (1480c1) [inline CMakeLists.txt]:
```suggestion
    if(CMAKE_C_COMPILER_ID MATCHES "GNU|Clang")
```

Comment (pudding8757) [reply CMakeLists.txt]: Done :)

**Comment by pudding8757** [comment]
CI failure on `GStreamer patch` seems to be unrelated to this PR. How can I re-run the CI?

**Comment by 1480c1** [comment]
Theoretically, you could close and reopen this PR, unsure if that will always rerun CI

**Comment by pudding8757** [comment]
> Theoretically, you could close and reopen this PR, unsure if that will always rerun CI

Forced-push could solve it !

**Pull request merged**